e2e/tests-dfx/ledger.bash

```diff
@@ -125,6 +157,18 @@ tc_to_num() {
     assert_match "Transfer sent at block height"
     assert_match "Refunded at block height"
     assert_match "with message: Provided subnet type type1 does not exist"
+    
+    # Transaction Deduplication
+    t=$(current_time_nanoseconds)
+
+    assert_command dfx ledger create-canister --amount=100 --created-at-time "$t" "$(dfx identity get-principal)"
+    assert_match "Transfer sent at block height"
```

Comment:
Since this PR adds output, we should account for that here.
```suggestion
    # shellcheck disable=SC2154
    assert_contains "Transfer sent at block height" "$stdout"
    # shellcheck disable=SC2154
    assert_contains "Using transfer at block height" "$stdout"
```